src/py42/modules/investigation.py

```diff
@@ -0,0 +1,7 @@
+class InvestigateModule(object):
```

Comment:
Let structure this this way:

`sdk.securitydata.savedsearches`
- `get()`
- `get_by_id()`
- `execute()`

i.e. add the the saved_search_client to `SecurityModule` instead of a new `InvestigateModule`.

Reply:
Moved.